WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -11,6 +11,7 @@ class NUXAbstractViewController: UIViewController {
     var helpBadge: WPNUXHelpBadgeLabel!
     var helpButton: UIButton!
     var loginFields = LoginFields()
+    var sourceTag = SupportSourceTag.generalLogin
```

Comment:
I had to give this one some thought, and I'd like to make a suggestion and see what you think. 

It seems like `sourceTag` would be a better fit as a part of the `SigninErrorViewControllerDelegate` protocol since those are the methods that use it.  I suggest moving the property there defined as a computed (not stored) property: `var sourceTag: String { get }` and have `NUXAbstractViewController` provide the default implementation. Other controllers could override to return their respective values.   This would keep sourceTag logically related to the protocol, and avoid needing to make an assignment in `viewDidLoad`. The value of the property never changes so making it immutable makes some sense to me.  

What do you think?